app/components/primer/alpha/tool_tip.ts

```diff
@@ -425,10 +353,30 @@ class ToolTipElement extends HTMLElement {
     if (!this.control) return
     if (!this.#allowUpdatePosition || !isPopoverOpen(this)) return
 
+    // Extract the value of the CSS variable (including the 'rem' unit)
+    // const remValueStr = getComputedStyle(document.documentElement).getPropertyValue('--overlay-offset').trim()
+    // // Use parseFloat to extract the numeric part of the value
+    // const remValue = parseFloat(remValueStr)
+
+    // // Get the base font-size of the document, assuming it's defined in pixels
+    // const baseFontSize = '16px'
+
+    // // Convert the 'rem' value to pixels
+    // const valueInPixels = remValue * baseFontSize
+
+    // // Use the converted value or fallback to 0 if conversion fails
+    // const finalValue = valueInPixels || 0
+
+    // Reading the CSS variable value
+    const tooltipOffset =
+      parseInt(getComputedStyle(document.documentElement).getPropertyValue('--overlay-offset'), 10) || 0
```

Comment:
`getComputedStyle` is a little bit icky because this code could potentially fail in various ways. I wonder if there's a better solution _just_ using CSS...


It seems as though the offset is either to the left, the right, the top or the bottom depending on the direction of the tooltip, so instead of this, how about something like the following CSS:


```css
:host(.tooltip-n, .tooltip-nw, .tooltip-ne) {
  --tooltip-top: calc(var(--tool-tip-position-top, 0) - var(--overlay-offset, 0));
  --tooltip-left: var(--tooltip-position-left)
}
:host(.tooltip-s, .tooltip-sw, .tooltip-se) {
  --tooltip-top: calc(var(--tool-tip-position-top, 0) + var(--overlay-offset, 0));
  --tooltip-left: var(--tooltip-position-left)
}
:host(.tooltip-w) {
  --tooltip-top: var(--tooltip-position-top);
  --tooltip-left: calc(var(--tool-tip-position-top, 0) - var(--overlay-offset, 0))
}
:host(.tooltip-e) {
  --tooltip-top: var(--tooltip-position-top);
  --tooltip-left: calc(var(--tool-tip-position-top, 0) + var(--overlay-offset, 0))
}
:host {
  inset: var(--tooltip-top, 0) auto auto var(--tooltip-left, 0) !important;
}
```